server/src/model.ts

```diff
@@ -610,14 +610,255 @@ export async function getAllRecentPublicActivities() {
     include: {
       owner: {
         select: {
-          email: true,
+          name: true,
         },
       },
     },
   });
   return docs;
 }
 
+// TODO - access control
```

Comment:
Remove these `TODO - access control` comments once you send in the `userId` and the verify that the user is an admin within these functions.